### PR TITLE
Updating documentation.

### PR DESCRIPTION
**Pull request by VardhanThigle**
Updating documentation for configuring network access, api enablement, and, quota errors.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -157,6 +157,30 @@ alias harbourbridge="go run github.com/GoogleCloudPlatform/spanner-migration-too
 
 This workflow also allows you to modify or customize the Spanner migration tool codebase.
 
+### Configuring connectivity to source database
+#### Configuring connectivity for `spanner-migration-tool`
+Ensure that the machine where you will run `spanner-migration-tool` is allowlisted to connect to the source database.
```
Comment (bharadwaj-aditya) [inline README.md:162]: nit: where you run

File: README.md [hunk 2]
```diff
@@ -157,6 +157,30 @@ alias harbourbridge="go run github.com/GoogleCloudPlatform/spanner-migration-too
...
+
+#### Configuring connectivity for data stream
+Follow the [Datastream guidelines](https://cloud.google.com/datastream/docs/configure-connecting-ip-allowlists) to allowlist datastream to access the source database.
+
+### API enablement
+Ensure that Datastream and Dataflow apis are enabled on your project.
+
+1. [Make sure that billing is enabled for your Google Cloud project](https://cloud.google.com/billing/docs/how-to/verify-billing-enabled#gcloud).
+2. Follow the [Datastream guidelines](https://cloud.google.com/datastream/docs/use-the-datastream-api#enable_the_api) to enable Datasteream api.
+3. Enable the Dataflow api by using:
```
Comment (bharadwaj-aditya) [inline README.md:180]: please add GCS here

File: README.md [hunk 3]
```diff
@@ -634,7 +658,12 @@ Spanner migration tool](#files-generated-by-spanner-migration-tool) contain deta
 about the schema and data conversion process, including issues and problems
 encountered.
 
-### 5. Reporting Issues
+### 5. Quota Issues
```
Comment (bharadwaj-aditya) [inline README.md:661]: nit: you may see quota exceeds errors.

File: README.md (hunk 2 above)
Comment (VardhanThigle) [reply README.md:180]: As discussed added a note for that.

**Review by VardhanThigle — COMMENTED** [review]
Thanks for the review. Incorporated the comments.

**Review by bharadwaj-aditya — APPROVED** [review]
LGTM

**Pull request merged**